### PR TITLE
Update what_is_citus.rst

### PR DESCRIPTION
**Pull request by ozgune**
Made a few small changes. In particular:

* Swapped the order of multi-tenant applications and real-time analytics, and
* Updated the multi-tenant description to reflect new features

@begriffs / @sumedhpathak -- what do you think?

### CODE REVIEW

File: aboutcitus/what_is_citus.rst [hunk 1]
```diff
@@ -10,7 +10,16 @@ Citus extends the underlying database rather than forking it, which gives develo
...
+Citus serves many use cases. Two common ones are scaling multi-tenant (B2B) databases and real-time analytics.
+
+Multi-Tenant Database
+~~~~~~~~~~~~~~~~~~~~~
+
+Most B2B applications already have the notion of a tenant, customer, or account built into their data model. In this model, the database serves many tenants, each of whose data is separate from other tenants.
+
+Citus provides full SQL coverage for this workload, and enables scaling out your relational database to 100K+ tenants. Citus also adds new features for multi-tenancy. For example, Citus supports tenant isolation to provide performance guarantees for large tenants, and has the concept of reference tables to reduce data duplication across tenants.
+
+These capabilities allow you to scale out your tenants data across many machines, and easily add more CPU, memory, and disk resources. Further, sharing the same database schema across multiple tenants makes efficient use of hardware resources and simplifies database management.
```
Comment (begriffs) [inline aboutcitus/what_is_citus.rst]: Add possessive: tenants' data

Comment (begriffs) [inline aboutcitus/what_is_citus.rst]: Also there's an inconsistent voice in these paragraphs, where some sentences talk directly to the reader ("Citus allows you to...") while others talk more generally about the properties of the system ("Citus supports tenant isolation...").

It's not a huge deal, just something I try to watch out for in my own writing.

Comment (begriffs) [inline aboutcitus/what_is_citus.rst]: @ozgune would you like me to take over this PR and make the edits?

Comment (ozgune) [inline aboutcitus/what_is_citus.rst]: @begriffs -- tenants' data makes sense.

On inconsistent voice, could you share an example revision you had in mind?

Comment (begriffs) [inline aboutcitus/what_is_citus.rst]: Actually the paragraph seems pretty good now that I read it again.

**Comment by begriffs** [comment]
This PR looks good to me.

**Pull request merged**